lucuma-schemas/src/clue/scala/lucuma/schemas/odb/SequenceQuery.scala

```diff
@@ -51,20 +48,12 @@ object SequenceSQL:
                   science {
                     ...gmosSouthSequenceFields
                   }
-                  setup {
```

Comment:
Setup was moved to digest?

Reply:
I just had to continue scrolling :D I see it in the digest.

Reply:
Yes